site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,49 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+
+#### Agones on GKE Autopilot
+
+Autopilot is GKE's fully-managed mode. GKE configures, maintains, scales, and
+upgrades nodes for you, which can reduce your maintenance and operating
+overhead. You only pay for the resources requested by your running Pods, and
+you don't pay for unused node capacity or Kubernetes system workloads.
+
+This section describes the Agones-specific considerations in Autopilot
+clusters. For a general comparison between Autopilot and Standard, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
+
+Autopilot nodes are, by default, optimized for most workloads. If you have
```

Comment:
🤔 this reads to me like compute classes give you far more control than I think they do?

I'd almost expect this sentence to read "Select GKE standard if you have specific hardware requirements, since you will have the most control over nodes, kernel parameters, etc".

I think this might be better as a constraint: 

* Limited hardware configuration: You may select one of the [three compute classes](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes#when-to-use), however this can be done per `GameServer` or `Fleet` configuration.

(or something like that) WDYT?

Reply:
You can select cpu architecture and set minimum CPU platforms as well! You can't select and use specific machine families or manually configure the underlying VMs though so I need to make that clear maybe?

Reply:
Maybe somewhere in between is the answer. 

Right now i read "specific hardware needs`, that reads to me like I can change anything. If I'm a game server developer I _definitely_ have specific hardware needs, and I know what they are, and they may or may not fit into the Autopilot model, and this currently reads (at least to me) like I can choose whatever I want -- which isn't the case.

I get three options 😄 that's it. We should be clear about it.

Reply:
I see what you mean, I'll do some fiddling tomorrow! Yeah I agree, we don't want to underplay what you can do with the compute classes but also don't want to imply that you get more flexibility than you actually do

Reply:
Ok, made some changes:

Autopilot nodes are, by default, optimized for most workloads. If some of your workloads have broad compute requirements such as Arm architecture or a minimum CPU platform, you can also choose a [compute class](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes) that meets that requirement. However, if you have specialized hardware needs that require fine-grained control over machine configuration, consider using GKE Standard.